### PR TITLE
Simplify script/modify_needle

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: script/modify_needle [hunk 1]
```diff
@@ -44,16 +44,15 @@ print help
 use strict;
 use warnings;
 
-use Mojo::JSON;    # booleans
+use Mojo::File qw(path);
+use Mojo::JSON;
```
Comment (Martchus) [inline script/modify_needle:48]: It doesn't look like that module is used at all.

**Review by Martchus — COMMENTED** [review]
Unfortunately lots of the `dump_templates` code isn't covered by tests.

File: script/modify_needle (hunk 1 above)
Comment (okurz) [reply script/modify_needle]: fixed

**Comment by okurz** [comment]
> Unfortunately lots of the `dump_templates` code isn't covered by tests.

Split out and moved to https://github.com/os-autoinst/openQA/pull/4023

**Review by Martchus — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Pull request merged**